cve_bin_tool/checkers/chrome.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2022 Orange
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+
+CVE checker for Chrome
+
+https://www.cvedetails.com/product/15031/Google-Chrome.html?vendor_id=1224
+
+"""
+
+from __future__ import annotations
+
+from cve_bin_tool.checkers import Checker
+
+
+class ChromeChecker(Checker):
+    CONTAINS_PATTERNS: list[str] = []
+    FILENAME_PATTERNS: list[str] = []
+    VERSION_PATTERNS: list[str] = []
```

Comment:
we also need some kind of version pattern in order to detect the version of product present in the particular binary.

Reply:
I was gonna talk to @terriko about that since the helper script didn't generate any version_patterns.
Probably have to use some other way based on the binaries downloaded

Reply:
In my experience the helper_script most of the time don't gives a version pattern, it is good quite with finding common(CONTAINS_PATTERNS) if you give more the 1 files to scan, heres something that can help :
[using strings and grep](https://github.com/intel/cve-bin-tool/blob/main/cve_bin_tool/checkers/README.md#finding-a-version-pattern)
with the checker you need to hit and trial with strings and grep, you can also look at the versions on the [cvedetails](https://www.cvedetails.com/product-list.php) to come up with a regex pattern for the version_patterns.